### ISSUE TITLE
Static test fails to prime the routers with more than one route

### ISSUE THREAD

**Issue by hueniverse** [delvedor/router-benchmark#5]
The test doesn't reflect real world use cases where there are more than one route, and routes overlap significantly. For example, adding this to the static test:

```js
const paths = ['/', '/a', '/bench', '/bench/ma', '/bench/marks', '/bench/mark/x', '/bench/mark', '/x'];
paths.forEach((path) => {

    findMyWay.on('GET', path, noop)
    routeRecognizer.add([{ path: path, handler: noop }])
    router.get(path, noop)
    serverRouter.route('GET', path, noop)
    koaRouter.get(path, noop)
    callRouter.add({ method: 'GET', path: path })
    express.route(path).get(() => { })
    barista.get(path, 'window.noop')
});
```

Changes the numbers dramatically:

```
find-my-way | lookup static route x 3,479,670 ops/sec ±1.46% (91 runs sampled)
call | lookup static route x 3,684,812 ops/sec ±1.12% (92 runs sampled)
route-recognizer | lookup static route x 315,841 ops/sec ±2.33% (84 runs sampled)
router | lookup static route x 141,319 ops/sec ±11.69% (37 runs sampled)
server-router | lookup static route x 138,788 ops/sec ±5.87% (60 runs sampled)
koa-router | lookup static route x 240,661 ops/sec ±5.78% (54 runs sampled)
routr | lookup static route x 638,345 ops/sec ±6.23% (72 runs sampled)
express | lookup static route x 125,066 ops/sec ±46.26% (34 runs sampled)
barista | lookup static route x 58,449 ops/sec ±5.74% (51 runs sampled)
```

**Comment by delvedor** [comment]
Thank you for your suggestions!
I'll update the script as soon as I can :)

**Comment by delvedor** [comment]
Suggestions addressed in #7 and #8, thank you for contributing! :)
